### PR TITLE
Add kconfig template for ABIs

### PR DESCRIPTION
**Pull request by oyvindronningstad**
Adapt bl_crypto and fix bl_crypto tests.

### CODE REVIEW

**Comment by oyvindronningstad** [comment]
cc @sigvartmh

**Comment by oyvindronningstad** [comment]
@sigvartmh any comments?

File: subsys/bootloader/bl_crypto/Kconfig [hunk 1]
```diff
@@ -99,4 +99,23 @@ config SB_PUBLIC_KEY_HASH_LEN
 	  The length to which public key hashes in the list of Root of Trust
 	  Public Keys are truncated.
 
+
```
Comment (sigvartmh) [inline subsys/bootloader/bl_crypto/Kconfig]: Intentional newline?

File: subsys/fw_metadata/Kconfig.template.fw_metadata_abi [hunk 2]
```diff
@@ -0,0 +1,19 @@
+
```
Comment (sigvartmh) [inline subsys/fw_metadata/Kconfig.template.fw_metadata_abi]: is it convention to leave this as newline?

File: subsys/bootloader/bl_crypto/Kconfig (hunk 1 above)
Comment (oyvindronningstad) [reply subsys/bootloader/bl_crypto/Kconfig]: No, removed

File: subsys/fw_metadata/Kconfig.template.fw_metadata_abi (hunk 2 above)
Comment (oyvindronningstad) [reply subsys/fw_metadata/Kconfig.template.fw_metadata_abi]: I don't know. I added a license instead :).

**Comment by oyvindronningstad** [comment]
@hakonfam merge?

**Pull request merged**